docs/securing_client_connection.rst

```diff
@@ -321,3 +321,37 @@ Using :class:`hazelcast.security.BasicTokenProvider` you can pass the given toke
     client = hazelcast.HazelcastClient(
         token_provider=token_provider
     )
+
+Kerberos Authentication
+-----------------------
+
+Python client supports Kerberos authentication with an external package.
+The package provides the necessary token provider that handles the
+authentication against the KDC (key distribution center) with the given
+credentials, receives and caches the ticket, and finally retrieves the token.
+
+You can install the package from PyPI.
+
+.. code:: bash
+
+    pip install hzkerberos
```

Comment:
The package is named `hazelcast-kerberos`.

Reply:
Oh, okay, fixed it in https://github.com/hazelcast/hazelcast-python-client/pull/453/commits/6a1eeafd687613a7b60712712fd3601fd8c926e9